README.md

```diff
@@ -1,24 +1,65 @@
-# README
+# DRAI
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+This is the production code for a tool designed for Assisters/Supervisors of community based organizations(CBOs), and will not be directly accessed by clients.
 
-Things you may want to cover:
+## Installation
 
-* Ruby version
+### Setup
 
-* System dependencies
+Run [`bin/setup`](bin/setup).
 
-* Configuration
+### Common Issues
 
-* Database creation
+-  If setting up Postgres.app, you will also need to add the binary to your path. e.g. Add to your `~/.zshrc`: `export PATH="$PATH:/Applications/Postgres.app/Contents/Versions/latest/bin"`
 
-* Database initialization
+#### Dependencies
 
-* How to run the test suite
+_You may already have these installed. Try running the **Setup** instructions above first._
 
-* Services (job queues, cache servers, search engines, etc.)
+1. [Homebrew](https://brew.sh/) for MacOS.
+2. A ruby version manager like [RVM](https://github.com/codeforamerica/howto/blob/master/Ruby.md) or rbenv.
+3. Locally installed Ruby of the version defined in `.ruby-version`. Because we are using an older version of Ruby, you may also need to run `gem update --system` before running `gem install bundler` and `bundle install`.
+4. Postgres 9.4+ `brew install postgres` or [Postgres.app](https://github.com/codeforamerica/howto/blob/master/PostgreSQL.md).
+5. Install Aptible CLI with Homebrew by running `brew cask install aptible`
+6. Install system dependencies defined in [Brewfile](Brewfile) with `brew bundle`
```

Comment:
There isn't a Brewfile, but you could make one and add the aptible cask to it.

Reply:
Brewfile made and added aptible cask to it